helm_deploy/values-train.yaml

```diff
@@ -16,6 +16,7 @@ generic-service:
     COMPONENT_API_URL: "http://not-used"
     DPS_URL: "https://digital-preprod.prison.service.justice.gov.uk"
     LOCATIONS_API_URL: "https://locations-inside-prison-api-train.hmpps.service.justice.gov.uk"
+    PRISON_API_URL: "https://prison-api-train.prison.service.justice.gov.uk"
```

Comment:
There is no train environment.  You will have mock this in the wiremock responses.